deployment/chainloop/templates/controlplane/deployment_sqlproxy.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if .Values.controlplane.sqlProxy.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "chainloop.sql-proxy.fullname" . }}
+  labels:
+    {{- include "chainloop.sql-proxy.labels" . | nindent 4 }}
+spec:
```

Comment:
the affinity code might look like

```yaml
 affinity:
    podAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
      - labelSelector:
          matchLabels:
            app.kubernetes.io/component: controlplane
```

but I think it will only guarantee that one of the controlplane replicas will have a proxy in the same node, not that both replicas have it. 

In any case I believe it is still worth it and we can just keep an eye on it. WDYT?